docker-compose.yml

```diff
@@ -632,11 +523,13 @@ services:
       PULUMI_CONFIG_PASSPHRASE: local-grapl-passphrase
       # Other environment variables like MG_ALPHAS are passed in via
       # Pulumi.local-grapl.yaml
-    depends_on:
-      localstack:
-        condition: service_healthy
-      kafka-broker:
-        condition: service_healthy
+    #    depends_on:
+    #      kafka-broker:
+    #        condition: service_healthy
```

Comment:
not sure if this should remain the case or not

Reply:
Since in the near future 
Kafka is being Nomad-provisioned before Pulumi gets run  (https://github.com/grapl-security/grapl/pull/1147 )
seems okayish for now

but in general, dead code bad

Reply:
Added kafka instructions to the BUILDING doc, so this is no longer necessary